code-challenges.test.js

```diff
@@ -49,4 +197,28 @@ const hand3 = [5, 5, 5, 5, 4]
 const hand4 = [7, 2, 7, 2, 7]
 // Expected output: true
 
+  
+
+
 // b) Create the function that makes the test pass.
+// To check full house
+const isThisFullHouse = (arr) => {
+    let countObj = {}
+    for(let x of arr){
+      countObj[x] = (countObj[x] || 0) + 1;
+    }
+    let vals = Object.values(countObj);
+    if((vals[0] === 2 && vals[1] === 3) || (vals[1] === 2 && vals[0] === 3)){
```

Comment:
I would make sure you mention this conditional statement in your pseudocode and how you are using it.